test/test_decomp.py

```diff
@@ -631,6 +631,41 @@ def test_batch_norm_unflatten_weight_bias(self, device):
         res = torch._decomp.decompositions.native_batch_norm(input, weight, bias, mean, var, False, 1, 1e-05)
         self.assertEqual(shape, res[0].shape)
 
+    def test_arange_graph(self, device):
+        from torch.fx.experimental.proxy_tensor import make_fx
+
+        def func(x, start):
+            le = x.shape[-1]
+            if start is None:
+                a = torch.arange(le, dtype=torch.float32, device=x.device)
+            else:
+                a = torch.arange(start, le, dtype=torch.float32, device=x.device)
+            return a
+
+        cfunc = make_fx(func, decomposition_table=decomposition_table)
+        fx_g = cfunc(torch.rand(10, device=device), None)
+        iota_str = "iota = torch.ops.prims.iota.default(10, start = 0, step = 1, " \
+            f"dtype = torch.int64, device = {repr(torch.device(device))}, requires_grad = False)"
+        self.assertExpectedInline(fx_g.code.strip(), f"""\
+def forward(self, x_1, start_1):
+    {iota_str}
```

Comment:
Oh sorry I didn't realize this was a parameterized test. Having the expected string be a format string like this wouldn't work with `EXPECTTEST_ACCEPT` which is the whole point (otherwise you would just use `assertEqual`).

The thing to do here is filter the divergent content out from the test string, so you could have a regex that removes `device=.*`.